### DISCUSSION TITLE
Other sorting options

### DISCUSSION THREAD

**Discussion by Stupidboat** [ente-io/ente#2780] [Enhancements]
Be able to sort ur entire library by filesize with the ability to filter photos or videos only, lacking from most popular photo hosts and would make it massively easier to clean up ur drive

**Comment by ua741** [comment]
In the mobile app, you can view large files here: Settings -> Backup -> Free up space -> Large files

**Comment by ua741** [comment]
Closing in favour of https://github.com/ente-io/ente/discussions/2807
